### DISCUSSION TITLE
Have you updated the mod .jar file yet? (it's missing some localization languages)

### DISCUSSION THREAD

**Discussion by RomanPlayer22** [cc-tweaked/CC-Tweaked#1374] [General]
Roughly how long will it take you to update the mod jar files to a newer version?
I have 100% translated the czech language file for the mod.

**Comment by SquidDev** [comment]
Thank you for your work on translations!

Releases kinda happen when they happen, there's not really a schedule to it. Probably this week, it depends how quickly some other projects comes together.

If you want to run the bleeding edge version, you can [grab it from the Artifacts section on CI](https://github.com/cc-tweaked/CC-Tweaked/actions/runs/4414321862).
